website/docs/react-native/santoku/application-architecture/push-notification/register-notification-channels.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 通知チャンネルの登録
+---
+
+Android8.0（APIレベル26）以降では、プッシュ通知はすべて通知チャンネルに割り当てる必要があります。通知チャンネルごとに、表示や鳴動動作などを適用します。
+
+このアプリでは、通知チャンネルを作成するライブラリとして[Notifee](https://notifee.app/)を使用します。
```

Comment:
Is there any reason why you chose Notifee instead of [Expo Notifications](https://docs.expo.dev/versions/latest/sdk/notifications/)?

Reply:
I take it back because the reason was stated elsewhere.